docs/userguide/configuration/providers/providers.rst

```diff
@@ -0,0 +1,72 @@
+Execution Providers
+===================
+
+Parsl presents a uniform interface to managing compute resources
+from a single computer with a single user up to
+supercomputers managed through batch schedulers.
+Each provider exposes three core actions: submit a
+job for execution (e.g., sbatch for the Slurm resource manager),
+retrieve the status of an allocation (e.g., squeue),
+and cancel a running job (e.g., scancel).
+
+All Providers manage resources in terms of **blocks**.
+Blocks are collections of one or more **nodes** that each
+run at least one worker. [#mpi]_
+Control the size of a block with the ``nodes_per_block`` option
+and the number of blocks with `elasticity settings <elasticity.html>`_.
+
+Start by selecting a scheduler from following classes,
+then configuring the details required.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Local Provider
+--------------
+
+The :class:`~parsl.providers.LocalProvider` starts workers on the same
+computer running the Parsl main script.
+It is primarily used when testing Parsl programs
+before deploying them to distributed resources.
+
+Cluster Schedulers
+------------------
+
+Schedulers are used at facilities where a single compute clusters shared by many teams.
+Users request individual "batch jobs" that will be ran once resources are available.
```

Comment:
- "... a single compute clusters shared ..." &rarr; "cluster is" ?
- "will be ran" &rarr; "will run" ?